_posts/2021-01-06-debezium-2020-recap.adoc

```diff
@@ -0,0 +1,84 @@
+
+First, some facts and numbers for you stats lovers out there:
+
++++<!-- more -->+++
+
+* After the release of link:/blog/2019/12/18/debezium-1-0-0-final-released/[Debezium 1.0] in December 2019, we successfully released a stable Debezium version at the end of each quarter, with preview releases roughly every three weeksfootnote:[ Where is Debezium 1.4, you ask? The agile bunch we are, we adhered to the "Individuals over processes" principle and decided to move this release to later this week, due to the holiday break :)]
+* About 1,400 commits in the core repo (plus many more in the other ones), 36 blog posts and release announcements, 166 threads on the https://groups.google.com/g/debezium/[mailing list] (if the query in my Google inbox is to be trusted)
+* About 100 new contributors, bringing the https://github.com/debezium/debezium/blob/master/COPYRIGHT.txt[overall number] of people contributing to the Debezium core repo to 245, plus additional people contributing to the other repositories of the Debezium GitHub organization
+* The first https://developers.redhat.com/blog/2020/04/14/capture-database-changes-with-debezium-apache-kafka-connectors/[GA release] of the commercially supported Debezium offering by Red Hat, as part of https://www.redhat.com/en/products/integration[Red Hat Integration]
+* link:/blog/2020/07/28/hello-debezium/[Two] link:/blog/2020/10/27/hello-debezium/[new] members on the core engineering team -- the more, the merrier!
+* About 1,600 additional GitHub ⭐s for the Debezium core repo, bringing the total number of star gazers to more than 4,100
+
+[.centered-image.responsive-image]
+====
+++++
+<img src="/assets/images/github_stars_2020.png" style="max-width:75%;" class="responsive-image">
+++++
+====
+
+While those figures give a nice impression of the overall activity of Debezium, they don't really tell _what_ has been happening exactly.
+What's behind the numbers?
+Here are some of my personal Debezium highlights from the last year:
+
+* Two new, community-led Debezium connectors for https://github.com/debezium/debezium-connector-db2/[Db2] and https://github.com/debezium/debezium-connector-vitess/[Vitess];
+a big shout-out to the engineers of link:/blog/2020/03/05/db2-cdc-approaches/[IBM] and link:/blog/2020/11/04/streaming-vitess-at-bolt/[Bolt], respectively, for stepping up and taking the lead of these connectors!
+* Besides these new connectors, each of the releases brought a wide range of new features; some of the things I'm most excited about are http://127.0.0.1:4000/documentation/reference/1.2/operations/debezium-server.html[Debezium Server] for integrating Debezium with message infrastructure like Apache Pulsar, AWS Kinesis, Google Cloud Pub/Sub, and Azure Event Hubs, the http://127.0.0.1:4000/documentation/reference/1.1/integrations/outbox.html[Quarkus extension] for implementing the outbox pattern, 
```

Comment:
The links to the documentation use `127.0.0.1:4000`.  These probably should be `link:/documentation/...`.

Reply:
Good spot, thx!